docs/source/models/cause.rst

```diff
@@ -558,6 +558,85 @@ below.
 Remission Rates
 ^^^^^^^^^^^^^^^
 
+**Definition**
+
+Remission is a measure of cases that recover from a with-condition state, given 
+a specified population and time period. Just as with incidence, the numerator is 
+given by the count of recovered (or *remitted*) cases, and the denominator is 
+the cumulative person-time [link to person-time definition in incidence 
+section] during which cases are able to go into remission.
+
+  For example, consider diarrhea cases in the Philippines in 2017. Say that that 
+  year, every such case remitted after an average of 5 days:
+
+  .. math::
+
+    \frac{\text{1 case}}{\text{5 person-days}}=\frac{365/5\text{ cases}}{\text{1 person-year}}=73\text{ cases/person-year}
+
+  This calculation is straightfoward, as diarrheal diseases have a highly
+  consistent disease duration.
+
+  In contrast, consider diabetes. Say that there were 142,794 prevalent cases of
+  diabetes (both type I and type II) in Moldova amongst males in 2017, and of 
+  those 142,794 cases, 509 remitted in 2017. This gives us the following rate:
+
+  .. math::
+
+    \frac{\text{509 cases}}{\text{142,794 person-years}} =
+    \frac{\text{0.0036 cases}}{\text{1 person-year}}
+
+  It is important to note that this is a remission rate for diabetes at all
+  ages. In the context of the diarrheal diseases example, this makes it seem as 
+  if diabetes cases remit, on average, after :math:`\frac{1}{0.0036}\simeq 279` 
+  years--which clearly does a poor job of capturing the behavior of diabetes. 
+  This sort of description was only appropriate for diarrhea, as there is a 
+  uniform remission rate across all cases. With diabetes, remission is not 
+  guaranteed for type II, and doesn't occur for type I, so remission rate does 
+  *not* tell us the average duration that an individual will experience diabetes.
+
+For both of the above rates, we think of :math:`r_1=0.2` as the probability that 
+a given case of diarrhea in the Philippies in 2017 will remitt in the next day, 
+and :math:`r_2=0.0036` as the probability that a given case of diabetes amongst 
+males in Moldova in 2017 will remitt in the next year. Notably, this is how 
+these rates are implemented in our models.
+
+.. todo::
+  Confirm "this is how these rates are implemented in our models."
+  is true
+
+Note that when we refer to remission rates, we are typically considering
+a rate within the infected or with-condition population. This is true both in
+general, and in the context of GBD--unlike with incidence, which GBD calculates 
+within the entire population, as discussed above.
+
+**Remission within GBD**
+
+Most nonfatal models in GBD are run using DisMod (:ref:`2017_cause_models`). DisMod 
+estimates compartmental models of disease, which includes an estimate of 
+remission in terms of **{remitted cases in the with-condition population} per 
+{person-year}**. Thus DisMod's estimates of various measures (prevalence, 
```

Comment:
I think it's more important to stress that the **person-years** are measured for the with-condition population. Remitted cases by definition have to come from the with-condition population, whereas person-years can be measured for different populations.

Reply:
Also, I think this definition (i.e. clarifying that remission uses person-time in the with-condition population) might make more sense in the previous paragraph where you first mention this.

Reply:
updating